src/Microsoft.AspNet.OData.Shared/Query/Expressions/ExpressionBinderBase.cs

```diff
@@ -192,7 +191,7 @@ internal Expression CreateBinaryExpression(BinaryOperatorKind binaryOperator, Ex
                     case BinaryOperatorKind.GreaterThanOrEqual:
                     case BinaryOperatorKind.LessThan:
                     case BinaryOperatorKind.LessThanOrEqual:
-                        left = Expression.Call(StringCompareMethodInfo, left, right, OrdinalStringComparisonConstant);
+                        left = Expression.Call(StringCompareMethodInfo, left, right);
```

Comment:
could it be a breaking change for non-ef core scenario?

Reply:
Sadly I can't negate this question, so there is a chance that a provider expects the ordinal case.